doc/api/test.md

```diff
@@ -316,6 +316,29 @@ Otherwise, the test is considered to be a failure. Test files must be
 executable by Node.js, but are not required to use the `node:test` module
 internally.
 
+## `runFiles([options])`
```

Comment:
bikeshedding: Would prefer if this was `run` rather than `runFiles` since it takes options (files is only one) and we might want to extend it to run other things in the future.